docs/contributing/docs_style_guide.md

```diff
@@ -10,6 +10,10 @@
 
 - Use the "bold/emphasis" style in Markdown by enclosing text in double asterisks or underscores, `**bold text**` or `__bold text__`, for UI elements that users will interact with. For example, a button label for a button that must be pressed should be made bold in Markdown.
 - Use the "italics" style in Markdown by enclosing text in single asterisks or underscores, `*italic text*` or `_italic text_`, for UI elements that have a label or title if you need to reference them in the documentation. For example, a title of a screen or page that will visit should be made italic in Markdown.
+- Use `>>` and `**bold text**` to indicate clicking through nested menu items, and also include the direct path. _Example:_
+```
+**Administration** >> **Structure** >> **Views** (~/admin/structure/views)
```

Comment:
I wouldn't include the tilde on the path as that has a very specific meaning when working on the command line and I think it could be confusing to new users.

Reply:
Good point. I'll fix that.